src/sass/layouts/_customer-reviews.scss

```diff
@@ -1,3 +1,185 @@
 .customer-reviews {
-  color: yellow;
+  margin-top: 80px;
+
+  @include for-tablet-up {
+    margin-top: 121px;
+  }
+
+  @include for-desktop-up {
+    margin-top: 123px;
+  }
+}
+.customer-reviews__image {
+  margin-bottom: 29px;
+  margin-left: auto;
+  margin-right: auto;
+
+  border-radius: 50%;
+
+  @include for-tablet-up {
+    margin-bottom: 19px;
+  }
+
+  @include for-desktop-up {
+    margin-bottom: 15px;
+  }
+}
+.customer-reviews__text-wrap {
+  min-width: 280px;
+  height: 308px;
+
+  margin-bottom: 30px;
+  padding: 44px 50px 25px 50px;
+
+  background-color: var(--slider-backg-color);
+
+  @include for-tablet-up {
+    min-width: 620px;
+    height: 164px;
+
+    margin-bottom: 24px;
+    padding: 30px 38px;
+  }
+
+  @include for-desktop-up {
+    min-width: 926px;
+    height: 138px;
+
+    margin-bottom: 15px;
+    padding: 32px 52px;
+  }
+}
+.customer-reviews__gradient-wrap {
+  position: relative;
+  width: 180px;
+  height: 248px;
```

Comment:
I think text should also be responsive in range 320-480. Only paddings should regulate it's position